.github/workflows/ci.yml

```diff
@@ -81,7 +81,7 @@ jobs:
           php-version: '8.0'
           tools: composer:v2
           coverage: none
-          extensions: decimal
+          extensions: none, curl, dom, filter, json, libxml, mbstring, openssl, pcre, phar, reflection, simplexml, spl, tokenizer, xml, xmlwriter
```

Comment:
I'd expect this not to be needed anymore?

Reply:
Actually this is desired. It defaults to having a bunch of extensions installed, but if you start the list with "none" it installs only the extensions you specify. This way we're not including any default-installed extensions that we don't want.

Reply:
cool, didn't know about that